api/common/v1alpha1/common_types.go

```diff
@@ -15,6 +15,20 @@ import (
 )
 
 const (
+	// TopologyRegionLabel is a label that can be applied to MachinePool, VolumePool, and BucketPool
+	//
```

Comment:
nitpick: can you remove the extra linebreak, this renders the package documentation a bit weird.